pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -208,15 +215,38 @@ func (b *BridgePodInterface) preparePodNetworkInterfaces() error {
 		return err
 	}
 
-	if err := b.createDefaultBridge(); err != nil {
+	if !delegateIp {
+		err = b.setupRouting()
```

Comment:
When do the routes disappear? Is it when we add the bridge or when we bring down eth0 to randomize the mac?

Reply:
When eth0 going down.